src/java.base/share/classes/java/lang/Class.java

```diff
@@ -1002,7 +1006,8 @@ public Module getModule() {
 
     private transient Object classData; // Set by VM
     private transient Object[] signers; // Read by VM, mutable
-    private final transient int modifiers;  // Set by the VM
+    private final transient char modifiers;  // Set by the VM
```

Comment:
Why the change of type here?

Reply:
I changed modifiers to u2 so that we won't have an alignment gap with the bool isPrimitiveType flag.